Source/ASCollectionNode.mm

```diff
@@ -1040,6 +1040,38 @@ - (void)updateCurrentRangeWithMode:(ASLayoutRangeMode)rangeMode;
   return result;
 }
 
+#pragma mark - UIGestureRecognizerDelegate Methods
+// The value returned below are default implementation of UIKit's UIGestureRecognizerDelegate
```

Comment:
Are we sure these are the default implementations of UICollectionView? Could we do some trickery to get UICollectionView's implementation here instead in case they change in the future?